### PR TITLE
Added iOS implementation

### PR DESCRIPTION
**Pull request by isnifer**

### CODE REVIEW

File: ios/TPSBarcode/TPSBarcode.m [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+    [self.session addOutput:captureMetadataOutput];
+    
+    [captureMetadataOutput setMetadataObjectsDelegate:self queue:self.queue];
+    [captureMetadataOutput setMetadataObjectTypes:[captureMetadataOutput availableMetadataObjectTypes]];
+    
+    self.previewLayer = [AVCaptureVideoPreviewLayer layerWithSession:self.session];
+    
+    [self.previewLayer setNeedsDisplayOnBoundsChange:YES];
+    [self.previewLayer setVideoGravity:AVLayerVideoGravityResizeAspectFill];
+}
```
Comment (densa) [inline ios/TPSBarcode/TPSBarcode.m]: [self.layer addSublayer:self.previewLayer];

File: ios/TPSBarcode/TPSBarcode.m [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+            }
+        }
+    }
+}
+
+- (void) layoutSubviews {
+    [super layoutSubviews];
+    [self.previewLayer setFrame:self.bounds];
+    [self setBackgroundColor:[UIColor blackColor]];
+    [self.layer insertSublayer:self.previewLayer atIndex:0];
```
Comment (densa) [inline ios/TPSBarcode/TPSBarcode.m]: Move into initCamera

**Comment by densa** [comment]
Looks good to merge

**Comment by cybergrind** [comment]
:+1:

**Comment by isnifer** [comment]
😃

**Pull request merged**